be/tool/beck.c

```diff
@@ -435,9 +435,9 @@ static struct gen g[MAX_GEN] = {};
 static bool  dry_run = false;
 static bool  signaled = false;
 
-#define FLOG(level, s)						\
-	M0_LOG(level, "        at offset: %"PRId64" errno: %s (%i), eof: %i", \
-	       ftell(s->s_file), strerror(errno), errno, feof(s->s_file))
+#define FLOG(level, rc, s)						\
+	M0_LOG(level, " rc=%d  at offset: %"PRId64" errno: %s (%i), eof: %i", \
+	       (rc), ftell(s->s_file), strerror(errno), errno, feof(s->s_file))
```

Comment:
Please remove the "strerror(errno), errno" from this FLOG.

They are incorrect.
I have raised in two other PR reviews.

Reply:
As per discussion with Nikita we have not removed existing errno related info as additional info will be always better than loosing debug data.

Reply:
@nikitadanilov  can you please comment ?

Reply:
I will approve this PR first.
If we decide to remove "errno" here, we can do it in another PR.